go.mod

```diff
@@ -20,26 +21,26 @@ require (
 	github.com/gorilla/websocket v1.4.2
 	github.com/hexops/gotextdiff v1.0.3
 	github.com/iancoleman/strcase v0.1.3
-	github.com/lucas-clemente/quic-go v0.20.0
+	github.com/lucas-clemente/quic-go v0.20.1
 	github.com/mattn/go-colorable v0.1.8
-	github.com/mattn/go-sqlite3 v1.14.6 // indirect
+	github.com/mattn/go-sqlite3 v1.14.7 // indirect
 	github.com/mgutz/ansi v0.0.0-20200706080929-d51e80ef957d // indirect
 	github.com/miekg/dns v1.1.41
-	github.com/montanaflynn/stats v0.6.5
-	github.com/ooni/probe-assets v0.0.0-20210401100648-90ed7b6dff90
-	github.com/ooni/psiphon v0.6.0
+	github.com/montanaflynn/stats v0.6.6
+	github.com/ooni/probe-assets v0.2.0
```

Comment:
We are now using semver for ooni/probe-assets. Not using it broke `go get -u -v ./...`.